### DISCUSSION TITLE
Captcha

### DISCUSSION THREAD

**Discussion by Jampang99** [engageub/InternetIncome#166] [General]
Can I please make a script to bypass AWS captca

**Comment by engageub** [comment]
Hello,
This discussion is not related to Internet Income Script.
Please focus on topics specifically pertaining to bandwidth sharing applications. 

Thank you.

**Reply by Jampang99** [reply]
If so, can I request 1 application similar to Honeygain, the application name is Uprock Earning

**Reply by engageub** [reply]
Hello,
Uprock requires one time verification to login each time. So it was not added.
In addition to that it is consuming more CPU. 
If you are aware of any docker image that exists please let me know. 

Thank you

**Comment by Jampang99** [comment]
![Screenshot_2024-11-15-10-26-46-00_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/user-attachments/assets/7eecdad4-dfbd-472d-b600-d942dba8d30e)



And what's wrong with this part

**Reply by engageub** [reply]
Could you please enable logs in properties file and look into tun2socks logs.
Also make sure your proxies in proxies.txt are in correct format.
Please make sure you are using the latest version. 
Also make sure you have enough resources. 
Thank you

**Comment by engageub** [comment]
Hello,
Alternatively you can use [test branch](https://github.com/engageub/InternetIncome/tree/test) and increase the delay between container using `DELAY_BETWEEN_CONTAINER` config variable and set  `USE_DOCKER_EMBEDDED_DNS=true` in properties.conf file if there are permission issues.


Thank you

**Comment by engageub** [comment]
Hello,
Were you able to resolve the issue?
If not, please create an issue in [Docker Forums](https://forums.docker.com/) and update the link here so that it would benefit others who have similar issues.

Thank you

**Comment by engageub** [comment]
Hi,

Uprock has been added to test branch. 
Please download the latest test branch using the following URL.

https://github.com/engageub/InternetIncome/tree/test

Thank you
